docs/en/observability/observability-ui.asciidoc

```diff
@@ -62,13 +62,13 @@ image::images/log-rate.png[Log rates]
 [[view-system-metrics]]
 == System metrics
 
-The *Metrics* chart provides you with a quick glance of the performance metrics
-aggregated on all of your running hosts. The metrics help you to evaluate the efficiency
-of hosts and determine whether there is any impact of resource consumption for your end-users.
+The *Metrics* chart provides you with a glance at your top hosts with the most significant
+resource footprint. The metrics help you evaluate the host efficiency and determine whether
+there is any impact of resource consumption for your end-users.
```

Comment:
"provides you with a glance at" seems wordy to me. What about something like this?

```suggestion
The *Metrics* chart displays your top hosts with the most significant
resource footprints. These metrics help you evaluate host efficiency and determine if
resource consumption is impacting end-users.
```

Reply:
Great suggestion. Thanks!